user-guide/Advanced_Modules/Automation_module/Using_CSharp/Building_interactive_Automation_scripts_with_CSharp.md

```diff
@@ -130,3 +130,6 @@ uib2.AppendBlock(blockButtonClose);
 // Display the dialog box
 engine.ShowUI(uib2);
 ```
+
+> [!NOTE]
```

Comment:
I'm not sure if this is the best place to add this note, as it appears to be relevant to more than just the example where you added it. Also, users might not notice it here. Perhaps we can move it to the top of the page, and add it as an "important" block below the note there?